vnets.tf

```diff
@@ -47,8 +47,9 @@ resource "azurerm_virtual_network" "public" {
 }
 
 ### Private VNet Address Plan:
-# - azure-net/vpn: 10.248.0.0/28 (x16 from 10.248.0.1 to 10.248.0.14)
-# - azure/privatek8s: 10.249.0.0/16 (x16 from 10.249.0.1 to 10.249.255.254)
+# - azure-net/dmz for external access (such as VPN external NIC): 10.248.0.0/28 (from 10.248.0.1 to 10.248.0.14). Defined below.
+# - Subnet "vpn-internal" for vpn VM internal NIC 10.248.0.64/28 (from 10.248.0.65 to 10.248.0.78) - defined in vpn.tf.
+# - Subnet "privatek8s" for the AKS cluster 10.249.0.0/16 (from 10.249.0.1 to 10.249.255.254) - Defined in jenkins-infra/azure/privatek8s.tf
```

Comment:
```suggestion
# - azure-net:vnets.tf/dmz = 10.248.0.0/28 (from 10.248.0.1 to 10.248.0.14), for external access (such as VPN external NIC)
# - azure-net:vpn.tf/vpn-internal = 10.248.0.64/28 (from 10.248.0.65 to 10.248.0.78),  for vpn VM internal NIC
# - azure:privatek8s.tf/privatek8s-tier = 10.249.0.0/16 (from 10.249.0.1 to 10.249.255.254), for the AKS cluster 
```
WDYT about this notation to indicate the location of the corresponding subnets?

Another alternative:
```suggestion
# - vnets.tf@azure-net/dmz = 10.248.0.0/28 (from 10.248.0.1 to 10.248.0.14), for external access (such as VPN external NIC)
# - vpn.tf@azure-net/vpn-internal = 10.248.0.64/28 (from 10.248.0.65 to 10.248.0.78),  for vpn VM internal NIC
# - privatek8s.tf@azure/privatek8s-tier = 10.249.0.0/16 (from 10.249.0.1 to 10.249.255.254), for the AKS cluster 
```
or omitting the .tf extension:
```suggestion
# - vnets@azure-net/dmz = 10.248.0.0/28 (from 10.248.0.1 to 10.248.0.14), for external access (such as VPN external NIC)
# - vpn@azure-net/vpn-internal = 10.248.0.64/28 (from 10.248.0.65 to 10.248.0.78),  for vpn VM internal NIC
# - privatek8s@azure/privatek8s-tier = 10.249.0.0/16 (from 10.249.0.1 to 10.249.255.254), for the AKS cluster 
```

In any case, I would put the description after the range, which is the most important thing you look at when you want to add an additional subnet.